toolkit/components/uniffi-bindgen-gecko-js/src/templates/js/Helpers.jsm

```diff
@@ -0,0 +1,83 @@
+// Write/Read data to/from an ArrayBuffer
```

Comment:
Not critical now, but at some point maybe we could move this to a regular JSM file in the source tree.  That way we could avoid repeating this code for each component.

Reply:
Yup agreed! I'll file a tiny follow up in our backlog 😄